course/migrations/0022_reportquestion.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.0.14 on 2021-10-08 03:56
+
```

Comment:
Only 1 migration file per PR per app. You can delete them and re-run the make migration command.

Reply:
I deleted the migrations and only added one to this PR now.